tests/unit/test_email_stats_source.py

```
@@ -0,0 +1,232 @@
+    sys.path.insert(0, project_root)
+
+from javdb.integrations.notify.email import service  # noqa: E402
+
+SPIDER_D1 = {'TotalProcessed': 11}
+SPIDER_LOCAL = {'TotalProcessed': 22}
+
+
+class _StubStatsRepo:
+    """Records which variant was called and returns a distinguishable row."""
+
+    calls: list[str] = []
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)